submissions/kuslosolodka/html-css-popup/style.css

```diff
@@ -0,0 +1,298 @@
+  margin: 0;
+  padding: 0;
+  font-family: "Raleway", sans-serif;
+  text-decoration: none;
+}
+body {
+  overflow: hidden;
+}
+.popup-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 30px 0;
+}
+#popup-btn {
+  display: none;
+}
+#popup-btn:checked ~ .popup-menu {
+  display: block;
+}
+.popup-icon {
+  width: 50px;
+  height: 50px;
+}
+.popup-icon:hover {
+  box-shadow: rgb(85, 91, 255) 0px 0px 0px 3px, rgb(31, 193, 27) 0px 0px 0px 6px,
+    rgb(255, 217, 19) 0px 0px 0px 9px, rgb(255, 156, 85) 0px 0px 0px 12px,
+    rgb(255, 85, 85) 0px 0px 0px 15px;
+}
+.popup-menu {
+  margin: 10px 0 30px 570px;
+  box-shadow: 3px 6px 16px 0px rgba(184, 180, 184, 1);
+  height: 600px;
+  width: 400px;
+  overflow-y: scroll;
+  border-radius: 10px;
+  display: none;
+}
+.icons {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+  margin: 30px 30px 30px 30px;
+}
+a {
+  display: block;
+  margin: 15px 20px 15px 10px;
+  align-items: center;
+  text-align: center;
+  color: black;
+}
+a:hover {
+  box-shadow: 3px 6px 16px 0px rgba(184, 180, 184, 1);
+  color: rgb(255, 157, 0);
+}
+a:focus {
+  color: rgb(255, 157, 0);
+  outline: none;
+  box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+}
+img {
+  width: 75px;
+  height: 75px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+.load-more {
+  cursor: pointer;
+  background: rgba(205, 203, 203, 0.397);
+  width: 383px;
+  height: 43px;
+  display: flex;
+  align-items: center;
+  border-radius: 2px;
+}
+.load-more:hover {
+  background: rgb(255, 157, 0);
+}
+#load-more-btn {
+  display: none;
+}
+#load-more-btn:checked ~ .hidden-icons {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  margin: 30px 30px 30px 30px;
+}
+.hidden-icons {
+  display: none;
+}
+p {
+  margin: 165px;
+  color: rgb(104, 102, 102);
+}
+.footer {
+  margin: 30px 0 -5px 0;
+}
+@media only screen and (max-width: 768px) {
+  .popup-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 30px 0;
+  }
+  .popup-menu {
+    margin: 0 0 30px 190px;
+    box-shadow: 3px 6px 16px 0px rgba(184, 180, 184, 1);
+    height: 500px;
+    width: 400px;
+    overflow-y: scroll;
+    border-radius: 10px;
+    display: none;
+  }
+  a {
+    display: block;
+    margin: 0 20px 5px 10px;
+    align-items: center;
+    text-align: center;
+    color: black;
+  }
+  .load-more {
+    cursor: pointer;
+    background: rgba(205, 203, 203, 0.397);
+    width: 400px;
+    height: 43px;
+    display: flex;
+    align-items: center;
+    border-radius: 2px;
+  }
+  p {
+    margin: 180px;
+    color: rgb(104, 102, 102);
+  }
+}
+@media only screen and (max-width: 600px) {
+  .popup-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 55px 0;
+  }
+  .popup-icon {
+    width: 60px;
+    height: 60px;
+  }
+  .popup-menu {
```

Comment:
Looks like something does not work properly on mobile
![image](https://user-images.githubusercontent.com/16196199/184924943-6c2988e5-6a59-4413-a170-e70c8915ac56.png)